src/App.tsx

```diff
@@ -1,109 +1,41 @@
 import React from 'react';
+import { PostList } from './components/PostList';
+import { Post } from './types/Post';
+import { User } from './types/User';
+import { Comment } from './types/Comment';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
 
-export const App: React.FC = () => (
-  <section className="App">
-    <h1 className="App__title">Static list of posts</h1>
-
-    <div className="PostList">
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">qui est esse</h3>
-
-          <p>
-            {' Posted by  '}
-
-            <a className="UserInfo" href="mailto:Sincere@april.biz">
-              Leanne Graham
-            </a>
-          </p>
-        </div>
-
-        <p className="PostInfo__body">
-          est rerum tempore vitae sequi sint nihil reprehenderit dolor beatae
-          ea dolores neque fugiat blanditiis voluptate porro vel nihil
-          molestiae ut reiciendis qui aperiam non debitis possimus qui neque
-          nisi nulla
-        </p>
-
-        <hr />
-
-        <b data-cy="NoCommentsMessage">No comments yet</b>
-      </div>
+const getUserById = (userId : number) : User | null => usersFromServer.find(
+  (user) => user.id === userId,
+) || null;
```

Comment:
In my opinion, smth like this is easer to read, but it's up to you
```suggestion
const getUserById = (userId: number): User | null => {
  return usersFromServer.find((user) => user.id === userId) || null;
};
```